public_html/wp-content/themes/wporg-events-2023/patterns/cover.php

```diff
@@ -0,0 +1,55 @@
+
+<?php
+/**
+ * Title: Front Page Cover
+ * Slug: wporg-events-2023/cover
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"className":"alignwide","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide"><!-- wp:columns {"verticalAlignment":null,"align":"wide","style":{"spacing":{"margin":{"bottom":"var:preset|spacing|40"}}}} -->
+<div class="wp-block-columns alignwide" style="margin-bottom:var(--wp--preset--spacing--40)"><!-- wp:column {"verticalAlignment":"bottom","width":"50%"} -->
+<div class="wp-block-column is-vertically-aligned-bottom" style="flex-basis:50%"><!-- wp:heading {"level":1,"style":{"elements":{"link":{"color":{"text":"var:preset|color|charcoal-0"}}}},"textColor":"charcoal-0"} -->
+<h1 class="wp-block-heading has-charcoal-0-color has-text-color has-link-color">Meet the community behind WordPress</h1>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"style":{"layout":{"selfStretch":"fit","flexSize":null},"elements":{"link":{"color":{"text":"var:preset|color|charcoal-0"}}}},"textColor":"charcoal-0"} -->
+<p class="has-charcoal-0-color has-text-color has-link-color">All over the world contributors meet to share, learn, and collaborate for the WordPress project.</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"50%"} -->
+<div class="wp-block-column" style="flex-basis:50%"><!-- wp:wporg/google-map {"id":"all-upcoming-map","apiKey":"WORDCAMP_DEV_GOOGLE_MAPS_API_KEY","filterSlug":"all-upcoming","showList":false,"showSearch":false} /--></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns -->
+
+<!-- wp:group {"align":"wide","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide"><!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"left":"var:preset|spacing|30"}}}} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"width":"33.34%","fontSize":"medium","fontFamily":"eb-garamond"} -->
+<div class="wp-block-column has-eb-garamond-font-family has-medium-font-size" style="flex-basis:33.34%"><!-- wp:group {"style":{"border":{"radius":"2px","width":"1px"},"spacing":{"padding":{"top":"var:preset|spacing|20","bottom":"var:preset|spacing|20","left":"var:preset|spacing|30","right":"var:preset|spacing|30"}}},"borderColor":"light-grey-1","layout":{"type":"constrained"}} -->
+<div class="wp-block-group has-border-color has-light-grey-1-border-color" style="border-width:1px;border-radius:2px;padding-top:var(--wp--preset--spacing--20);padding-right:var(--wp--preset--spacing--30);padding-bottom:var(--wp--preset--spacing--20);padding-left:var(--wp--preset--spacing--30)"><!-- wp:paragraph {"style":{"elements":{"link":{"color":{"text":"var:preset|color|charcoal-0"}}}},"textColor":"charcoal-0"} -->
+<p class="has-charcoal-0-color has-text-color has-link-color">44,744 events this year</p>
```

Comment:
```suggestion
<p class="has-charcoal-0-color has-text-color has-link-color">3,925 events this year</p>
```

Merges https://github.com/WordPress/wordcamp.org/commit/f626c4c27da13df3476d3b4b04cf1ff6eed944d2

Reply:
It looks like https://github.com/WordPress/wordcamp.org/pull/1117 is also missing. That may be difficult to rebase given the way Gutenberg stores things. It's totally fine with me if you want to change anything I did in that commit, I just want to make sure you're aware of it.

Reply:
Yep, It'll force me to rebase this before merging. It'll catch the drift.

Reply:
A sorry missed the context, I hadn't realized you made style changes. Yeah, I had to overwrite some things because the merge conflicts were pretty tricky. If you had figured out a better or more extensible way in that commit that isn't represented here, make that change. There are so many different ways to handle the mobile/tablet/desktop layout changes and I'm not particular to any specific one.

Reply:
👍🏻